### ISSUE TITLE
[api-extractor] InternalError when analyzing EditorState from draft-js 

### ISSUE THREAD

**Issue by pgonzal** [Microsoft/web-build-tools#1001]
@nvh reported an error in issue https://github.com/Microsoft/web-build-tools/issues/949, which on investigation turned out to be a separate issue:

> I encountered this error importing the `EditorState` from `draft-js` and using it in a class
> A minimal reproduction looks like this:
> ```typescript
> import { EditorState } from "draft-js"
> export class Text {
>     editorState: EditorState
> }
> ```
> making the editorState private fixes the problem. 
> Example project: 
> [registerd-as-non-imported.zip](https://github.com/Microsoft/web-build-tools/files/2633832/registerd-as-non-imported.zip)
> Reproduction steps:
> <img width="672" alt="screenshot 2018-11-30 at 16 05 03" src="https://user-images.githubusercontent.com/248861/49296940-ca99bc00-f4b9-11e8-825c-c9d6152f0b12.png">
> 

The error message was:
```
The symbol EditorState is being imported after it was already registered as non-imported
```

**Comment by pgonzal** [comment]
Inside the typings for **@types\draft-js\index.d.ts** we have this:

```ts
export as namespace Draft;
declare namespace Draft {
    namespace Model {
        namespace ImmutableData {
            class EditorState extends Record {
                // ...
            }
        }
    }
}

import EditorState = Draft.Model.ImmutableData.EditorState;

export {
    EditorState
};
```

Initially I thought this was another instance of the `export { x }` construct (issue #663), however it seems to be a more mundane problem.  In [AstSymbolTable.ts](https://github.com/Microsoft/web-build-tools/blob/ee668114347e573a4c7de1c266e9eb86682a481c/apps/api-extractor/src/analyzer/AstSymbolTable.ts#L308), the compiler's `isSourceFileFromExternalLibrary()` API is returning `false` for **@types\draft-js\index.d.ts** which causes a full analysis of this unsupported construct.

```ts
        // If the file is from a package that does not support AEDoc, then we process the
        // symbol itself, but we don't attempt to process any parent/children of it.
        const followedSymbolSourceFile: ts.SourceFile = followedSymbol.declarations[0].getSourceFile();
        if (this._program.isSourceFileFromExternalLibrary(followedSymbolSourceFile)) {
          if (!this._packageMetadataManager.isAedocSupportedFor(followedSymbolSourceFile.fileName)) {
            nominal = true;
          }
        }
```

Whereas if `nominal = true` then the problematic construct would not be analyzed and the API extraction would be successful.

It seems that the compiler's `isSourceFileFromExternalLibrary()` doesn't do what I thought it did, since **@types/draft-js** sure seems like an "external library".  Perhaps we need to revert to the earlier approach relying on [isAedocSupportedFor](https://github.com/Microsoft/web-build-tools/blob/f8bcf24906f5bd057dbf87cf562c54877abad4e2/apps/api-extractor/src/generators/dtsRollup/AstSymbolTable.ts#L316) --> [PackageJsonLookup](https://github.com/Microsoft/web-build-tools/blob/f8bcf24906f5bd057dbf87cf562c54877abad4e2/apps/api-extractor/src/generators/dtsRollup/PackageMetadataManager.ts#L69) to determine whether a project belongs to the current project or not...  This API was introduced in PR https://github.com/Microsoft/TypeScript/pull/12054

@DanielRosenwasser FYI

**Comment by pgonzal** [comment]
This is working now in my branch for PR https://github.com/Microsoft/web-build-tools/pull/1002

It makes the analysis accurate so that we can simply test whether `followAliasesResult.astImport` is defined or not.  If so, then the symbol is from an external package.

**Comment by pgonzal** [comment]
Fixed in #1002
